assets/src/edit-story/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -262,7 +260,99 @@ describe('Right Click Menu integration', () => {
     });
   });
 
-  describe('right click menu: page/background', () => {
+  describe('right click menu: foreground and background media actions', () => {
```

Comment:
updated these two tests to check foreground and background media - that way we don't have to have separate tests for each one.